src/NuGet.Core/NuGet.PackageManagement/Utility/UriHelper.cs

```diff
@@ -107,7 +107,7 @@ public static bool IsHttpSource(string source, PackageSourceProvider packageSour
                 }
 
                 var packageSource = packageSourceProvider.LoadPackageSources()
-                    .FirstOrDefault(p => p.Name.Equals(source, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
This `IsHttpSource` method not used anywhere in our code, maybe we need to remove it.